haystack/nodes/retriever/dense.py

```diff
@@ -1898,3 +1898,303 @@ def save(self, save_dir: Union[Path, str]) -> None:
         :type save_dir: Union[Path, str]
         """
         self.embedding_encoder.save(save_dir=save_dir)
+
+
+class MultihopDenseRetriever(EmbeddingRetriever):
```

Comment:
I'd propose renaming this to `MultihopEmbeddingRetriever`, this might make it more clear that this is an extension of the `EmbeddingRetriever`.

Reply:
Done